docs/modules/reference/pages/configuration/file-editor.adoc

```diff
@@ -0,0 +1,58 @@
+[[file-editor]]
+= File Editor: Manage Configuration Files from the UI
+
+The File Editor, available in the new UI, allow users to view and edit configuration files (`.xml`, `.cfg`, `.drl`, etc.) with the OpenNMS UI instead of having to edit them directly in the file system (for example in the `/etc` directory).
+
+NOTE: Making changes to configuration files will affect how OpenNMS operates and incorrectly editing a file could result in features no longer working or not working as intended.
+
+To access the File Editor, go to the new UI, for example by clicking on "UI Preview", then navigating to "File Editor".
+
+== Viewing Files
+
+After navigating to the File Editor, a tree view of the OpenNMS `/etc` directory is displayed.
+
+From here you can:
+
+- navigate to any file
+- expand and collapse subtrees
+- search for any part of a file or directory name
+- view and edit files
+
+.File Editor main screen
+image::configuration/file-editor-main.png[]
+
+## Editing Files
+
+To edit a file, simply start typing within the file text display.
+
+To save the file, click "Save".
+
+CAUTION: This will overwrite the file on disk, there is no versioning or backup file support at this time.
+
+Use the "Reset" button to reload the file from disk into the editor - this will be the text as it was last saved to disk, not necessarily the original text before you began any editing.
```

Comment:
```suggestion
Click the *Reset* button to reload the file from disk into the editor. 
This will be the text as it was last saved to disk, not necessarily the original text before you began editing.
```

Reply:
Fixed